wordpress-mobile/WordPress-iOS
# My Site Dashboard (Phase 2): Quick Action card UI

Part of #17874 

## Description
- Adds the initial UI for the Quick Actions card on the dashboard

## Notes
- Changes are behind a feature flag
- Quick Actions cards aren't hooked up yet -- tapping won't do anything

## How to test

Make sure to run your sandbox with the 1.1 API diff applied and to have the My Site Dashboard flag enabled:

1. Open the app
2. Go to My Site
3. Switch to the dashboard
4. ✅  Quick Actions cards are displayed horizontally and can be scrolled

Light | Dark
-- | --
<img src ="https://user-images.githubusercontent.com/6711616/153433533-8aac8d6e-a927-4d03-9eb6-7a43ed1e2ec9.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/153433537-75d4992d-19c4-4c2a-ae8d-45a07c50e2b5.png" width=200>


https://user-images.githubusercontent.com/6711616/153433504-6d44ccec-a778-4f3b-8553-326eae178b60.mp4


## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
